Source/Relativity.Test.Helpers/TestHelpersKepler/TestHelpersKepler.Interfaces/TestHelpersModule/v1/ITestHelpersService.cs

```diff
@@ -38,5 +38,9 @@ public interface ITestHelpersService : IDisposable
 		[HttpPost]
 		[Route("GetFieldCount")]
 		Task<FieldCountResponseModel> GetFieldCountAsync(int fieldArtifactId, int workspaceId);
+
+		[HttpPost]
+		[Route("GetFolderName")]
```

Comment:
why are we having a different custom route other than the method name itself?

Reply:
Good question